tests/backend/test_bosonic_backend.py

```diff
@@ -474,6 +474,35 @@ def test_gkp_complex(self):
         with pytest.raises(NotImplementedError):
             backend.run_prog(prog)
 
+    @pytest.mark.parametrize("eps", EPS_VALS)
+    def test_sensor_gkp(self, eps):
+        r"""Checks that the GKP sensor state is invariant under Fourier."""
+        x = np.linspace(-3 * np.sqrt(np.pi), 3 * np.sqrt(np.pi), 40)
+        p = np.copy(x)
+
+        # Prepare GKP 0 and apply Hadamard
+        prog_sensor = sf.Program(1)
+        with prog_sensor.context as q:
+            sf.ops.GKP(state=[np.pi / 2, 0], epsilon=eps, shape="rectangular", alpha=2) | q[0]
+
+        backend_sensor = bosonic.BosonicBackend()
+        backend_sensor.run_prog(prog_sensor)
+        state_sensor = backend_sensor.state()
+        wigner_sensor = state_sensor.wigner(0, x, p)
+
+        # Prepare GKP +
+        prog_sensor_F = sf.Program(1)
+        with prog_sensor_F.context as qF:
+            sf.ops.GKP(state=[np.pi / 2, 0], epsilon=eps, shape="rectangular", alpha=2) | qF[0]
+            sf.ops.Rgate(np.pi / 2) | qF[0]
+
+        backend_sensor_F = bosonic.BosonicBackend()
+        backend_sensor_F.run_prog(prog_sensor_F)
+        state_sensor_F = backend_sensor_F.state()
+        wigner_sensor_F = state_sensor_F.wigner(0, x, p)
+
+        assert np.allclose(wigner_sensor, wigner_sensor_F)
```

Comment:
I don't understand what you mean? I guess we don't want to check if they are exactly equal, since they likely do differ at some decimal point since one is a linear combination of Gaussians and the other is built from Fock state Wigner functions.

Reply:
I mean that when using `allclose` one may need to use non-default tolerances to ensure that the comparison is not too relaxed or restrictive. This may require some trial and error.